### PR TITLE
Fix open button position

### PR DESCRIPTION
**Pull request by FadhlanR**
https://github.com/cardstack/boxel/assets/12637010/8f1234ba-8352-4b35-9d82-9aae74b869ec

- [x] Fix the "Open" position button.
- [x] Fix the "Open" button z-index problem. The fix requires some refactoring so we can put an overlay button in each card.

### CODE REVIEW

**Review by jurgenwerk — COMMENTED** [review]
Will this be running overlay refresh calculation loop for every item in the stack?

**Comment by lukemelia** [comment]
Is it time yet to consider a dedicated library for this like https://popper.js.org/ ?

**Comment by FadhlanR** [comment]
@jurgenwerk I think so.

**Comment by FadhlanR** [comment]
@lukemelia I decided to use `ember-velcro`. https://github.com/cardstack/boxel/pull/429/commits/5e8f1dea0200ea269633a29c3cf54f1ba9ac6233

File: packages/host/app/components/operator-mode-stack-item.gts [hunk 1]
```diff
@@ -0,0 +1,234 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { Card, CardContext } from 'https://cardstack.com/base/card-api';
+import Preview from './preview';
+import { fn, array } from '@ember/helper';
+import type CardService from '../services/card-service';
+// import getValueFromWeakMap from '../helpers/get-value-from-weakmap';
```
Comment (habdelra) [inline packages/host/app/components/operator-mode-stack-item.gts]: probably unnecessary comment

**Review by lukemelia — APPROVED** [review]
LGTM

**Review by burieberry — APPROVED** [review]
Looks good

**Pull request merged**